spec/models/resource_action_workflow_spec.rb

```diff
@@ -53,9 +53,12 @@
       end
     end
 
-    context "#submit_request" do
+    describe "#submit_request" do
       subject { ResourceActionWorkflow.new({}, admin, resource_action, :target => target) }
       let(:resource_action) { @resource_action }
+      let(:task) { double }
+
+      before { allow(task).to receive(:id).and_return(0) }
```

Comment:
Minor, but I think we should have the test return a non-zero value which is more inline with what we would expect from the db.